ee/clickhouse/sql/events.py

```diff
@@ -98,6 +98,35 @@
     database=settings.CLICKHOUSE_DATABASE,
 )
 
+KAFKA_EVENTS_TABLE_JSON_SQL = lambda: EVENTS_TABLE_BASE_SQL.format(
+    table_name="kafka_json_events",
+    cluster=settings.CLICKHOUSE_CLUSTER,
+    engine=kafka_engine(topic=KAFKA_EVENTS_JSON),
```

Comment:
The variable name and topic name don't agree - should we fix it? I guess not if others are similarly out of sync.

Reply:
this follows the convention here